src/Symfony/Component/Serializer/Normalizer/FormErrorNormalizer.php

```diff
@@ -38,6 +38,13 @@ public function normalize(mixed $object, string $format = null, array $context =
         return $data;
     }
 
+    public function getSupportedTypes(?string $format): array
+    {
+        return [
+            FormInterface::class => false,
```

Comment:
`__CLASS__ === static::class`

Reply:
Are you sure this is cacheable?
`supportsNormalization` return value depends on the data so I'm not sure this is the case.